src/cloud/indexer/md/mapping.o.go

```diff
@@ -106,7 +106,7 @@ const IndexMapping = `
         },
         "tokenizer": {
           "ngram_tokenizer": {
-            "type": "nGram",
+            "type": "ngram",
```

Comment:
This was renamed in v8 ([docs](https://www.elastic.co/guide/en/elasticsearch/reference/8.18/migrating-8.0.html)).